Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml

```
@@ -457,6 +457,107 @@
   - type: WirelessNetworkConnection
     range: 1200
 
+- type: entity
+  parent: BaseComputerAiAccess
+  id: ComputerStationTeleportersControl
+  name: station teleporters control console
+  description: Used to monitor station teleport system operations.
+  components:
+  - type: Sprite
+    layers:
+    - map: ["computerLayerBody"]
+      state: computer
+    - map: ["computerLayerKeyboard"]
+      state: generic_keyboard
+    - map: ["computerLayerScreen"]
+      state: teleporters_station
+    - map: ["computerLayerKeys"]
+      state: telesci_key
```

```suggestion
    - map: ["computerLayerKeys"]
      state: telesci_key
    - map: [ "enum.WiresVisualLayers.MaintenancePanel" ]
      state: generic_panel_open
```
wire panel is missing